src/scss/partials/_chatroom.scss

```diff
@@ -0,0 +1,128 @@
+@use "./variables" as *;
+
+.chatWindow{
+    border: 1px solid black;
+    width: 500px;
+    height: 80vh;
+    margin: 2% auto 0 auto;
+    overflow: auto;
+    position: relative;
+    width: min(50%,120rem);
+    border-radius: 2rem 2rem 0 0 ;
+    box-shadow: $boxShadowConfig1;
+    min-width: 320px;
+  }
+
+  .messageForm{
+    width: min(50%,120rem);
+      margin: 0 auto;
+      @include flex(row, flex-start,flex-start,nowrap);
+      min-width: 320px;
+    label{
+      display: none;
+    }
+    button{
+      padding: 1rem;
+      margin: 0;
+      border-radius: 0 0 2rem 0;
+    }
+    input{
+      flex-grow: 1;
+      padding: 1rem;
+      border-radius: 0 0 0 2rem;
+    }
+  }
+  
+  .msgBody{
+    padding: 1%;
+    position: relative;
+    
+    //need a margin at the bottom
+    margin: 5px;
+
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+
+    .msgFlexParent{
+      @include flex(row,flex-start,center,nowrap);
+    }
+
```

Comment:
loved the Names of users on top along with the unique 
I think a padding for the right user name would make it a little 
more visible